docs/source/events.rst

```diff
@@ -422,30 +529,809 @@ bar icon.
 - Payload of start event: ``NULL``
 - Payload of stop event: ``NULL``
 
+Desktop Searches
+~~~~~~~~~~~~~~~~
 
-Test Events
------------
+Removed in 4.0.0.
 
-The following are only ever used in tests, and are thus discarded by the server
-upon reception.
+We record searches made from the desktop search bar.
 
-We document them here to make sure they don't get reused inadvertently by real
-events.
+- Since: 2.1.2
+- UUID: ``b02266bc-b010-44b2-ae0f-8f116ffa50eb``
+- UUID name: ``EVENT_DESKTOP_SEARCH`` in gnome-shell
+- Payload: type ``(us)``
 
-- `Smoke tests <https://github.com/endlessm/eos-metrics/blob/ab66c7319c573999740f636555b14b6f658e82c0/tests/smoke-tests/smokeLibrary.js#L22-L27>`_
+  - search provider
 
-  - ``72fea371-15d1-401d-8a40-c47f379f64fd``
-  - ``9a0cf836-12cd-4887-95d8-e48ccdf6e552``
-  - ``b1f87a3f-a464-48d4-8e35-35dd45659010``
-  - ``b2b17dfd-c30e-4789-abcc-4a38323127f6``
-  - ``b89f9a4a-3035-4fc3-9bef-584367fe2c96``
-  - ``fb59199e-5384-472e-af1e-00b7a419d5c2``
+    - local: 0
+    - Google: 1
 
-- `Event recorder tests <https://github.com/endlessm/eos-metrics/blob/ab66c7319c573999740f636555b14b6f658e82c0/tests/test-event-recorder.c#L29-L30>`_
+  - query string (what the user searched for)
 
-  - ``350ac4ff-3026-4c25-9e7e-e8103b4fd5d8``
-  - ``d936cd5c-08de-4d4e-8a87-8df1f4a33cba``
+.. note::
 
-- `Daemon tests <https://github.com/endlessm/eos-event-recorder-daemon/blob/efc6bb0e1283236ee4fe9c3d7fc992c4a53436d8/tests/daemon/test-daemon.c#L51>`_
+   Since the 2.1.6. release, Google searches have no longer been recorded.
 
-  - ``350ac4ff-3026-4c25-9e7e-e8103b4fd5d8``
+Knowledge App Search
+~~~~~~~~~~~~~~~~~~~~
+
+Removed in 4.0.0.
```

Comment:
I'd prefer "not stored since Endless OS 4.0.0". We are unlikely to update the 6 different versions of the Endless SDK (Flatpak) which sends this, but instead just drop it on receipt.

(In fact we never have extracted this event; they're all sitting in the unknown_singular_event table in raw gvariant form.)